pkgs/development/libraries/postgis/default.nix

```diff
@@ -89,6 +89,10 @@ in rec {
     sha256 = "02gsi1cm63kf0r7881444lrkzdjqhhpz9a5zav3al0q24nq01r8g";
     sql_srcs = ["postgis.sql" "spatial_ref_sys.sql"];
     builtInputs = [gdal json_c pkgconfig];
+
+    # postgis config directory assumes /include /lib from the same root for json-c library
+    NIX_LDFLAGS="-L${json_c.out}/lib";
```

Comment:
I recommend using `getLib` here instead, it will be robust to future changes to the outputs. That is, do `NIX_LDFLAGS = "-L${stdenv.lib.getLib json_c}"` instead.